cmd/up.go

```diff
@@ -28,6 +37,50 @@ var (
 				os.Exit(ExitSetupFailed)
 			}
 
+			log.Printf("Connecting to management server %s", config.ManagementAddr)
```

Comment:
Not sure if it is too early for this PR and you have this planned, but just a couple of ideas:

1. I think we could use `mgmCtx, cancel := context.WithTimeout(context.Background(), 5*time.Second)` to throw an error if it takes a long time to connect to teh server
2. Detect whether the service has TLS enabled (probably take this from the config file) and create a client with credentials. The client creation of the service with TLS could look like this:

``` 
mgmCtx, cancel := context.WithTimeout(context.Background(), 5*time.Second)  
mgmConn, err := grpc.DialContext(  
		mgmCtx,  
		config.ManagementAddr,  
		grpc.WithTransportCredentials(credentials.NewTLS(tlsConf)),  
		grpc.WithBlock(),  
		grpc.WithKeepaliveParams(keepalive.ClientParameters{  
			Time:    3 * time.Second,  
			Timeout: 2 * time.Second,  
		}))
```

Reply:
1. Done.
2. Better create a separate PR for this.

Reply:
Soudns good!